### ISSUE TITLE
Incorrect docs for Gaussian Dropout layer

### ISSUE THREAD

**Issue by arodiss** [keras-team/keras#8587]
Description of Gaussian Dropout layer at https://keras.io/layers/noise/ is copy-paste from Gaussian Noise layer, which is incorrect
